docs/README-Extending.md

```diff
@@ -168,6 +168,37 @@ The FTP Zowe Explorer extension provides examples for providing a data provider
 
 These are parallel implementations of the same operations that are provided by Zowe Explorer itself using the z/OSMF interaction protocol. You can find that implementation for reference in the file [packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts](../packages/zowe-explorer-api/src/profiles/ZoweExplorerZosmfApi.ts).
 
+## Using the Zowe Explorer ProfilesCache for their own unrelated profiles
+
+In the previous two sections we outlined how extenders can access the cached profiles of Zowe Explorer and can provide new profile types for a new data providers for any of the three Zowe Explorer tree views. Another use case would be that a Zowe Explorer extender does not add a new data provider to Zowe Explorer, but instead an all out new fourth (or more) tree view(s) showing data from a different data source that is not data sets, USS, or JES. The extender would still want to use the same ProfilesCache than Zowe Explorer to be able to react to the same refresh operations, i.e. when the user clicks the Refresh Explorer button in any of the tree view that all profiles including the custom ones should be reloaded.
+
+To support that the Zowe Explorer profiles cache that you can access as described above offers register method that allows adding a profile type to the cache that is not associated with any of the three APIs list above. Note, that the profile type must be valid Zowe CLI profile that is installed on the end user's home directory. The extension needs to therefore make sure it called the `initForZowe()` before trying to register a custom type.
+
+```typescript
+// Retrieve the Zowe Explorer API object from the currently running instance.
+// It must be at least Zowe Explorer 1.18.0 or newer or undefined will returned.
+const zoweExplorerApi = ZoweVsCodeExtension.getZoweExplorerApi("1.18.0");
+if (zoweExplorerApi) {
+  // Initialized the users ~/.zowe directory with the metadata for FTP profiles in case
```

Comment:
Not sure it matters too much, but this documentation references both CICS and FTP things - I know it's only an example section but might be worth a look over??

Reply:
Good catch!